public/romania/manifest.json

```diff
@@ -7,50 +7,68 @@
   "theme_color": "#b80000",
   "icons": [
     {
-      "src": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/romania/images/icons/icon-72x72.png?v=1",
+      "src": "/romania/images/icons/icon-72x72.png",
```

Comment:
I think these will still need the full static assets path name: `https://static.files.bbci.co.uk/ws/simorgh-assets/public/`. Just looking at Dari for example, it has the full path referenced.